atmel_sam4s_xplained_pro_config

```diff
@@ -0,0 +1,1546 @@
+#
```

Comment:
@yjdwbj Looks like the file snuck in.  Can you remove it.

Reply:
You mean the `arch/arm/src/sam34/Kconfig`  file ?  But I'm not changed this file,What you mean the other file?